<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget1" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget1 {background-color: rgb(147, 205, 255);};

</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 239, 115);
color: rgb(0, 0, 0);
font: 75 40pt &quot;Product Sans&quot;;
font-weight: bold;
</string>
    </property>
    <property name="text">
     <string>RSA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 15pt &quot;Product Sans&quot;;</string>
    </property>
    <property name="text">
     <string>n Key</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 15pt &quot;Product Sans&quot;;</string>
    </property>
    <property name="text">
     <string>e (public)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 15pt &quot;Product Sans&quot;;</string>
    </property>
    <property name="text">
     <string>d (private)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="loadEKey">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>180</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 500 10pt &quot;Product Sans&quot;;
border-radius: 10px;
background-color: rgb(255, 239, 115);


</string>
    </property>
    <property name="text">
     <string>Load Key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadDKey">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>240</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 500 10pt &quot;Product Sans&quot;;
border-radius: 10px;
background-color: rgb(255, 239, 115);


</string>
    </property>
    <property name="text">
     <string>Load Key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateKeyButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>181</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 500 15pt &quot;Product Sans&quot;;
border-radius: 10px;
background-color: rgb(255, 239, 115);


</string>
    </property>
    <property name="text">
     <string>Generate Key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveKeyButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>290</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 500 15pt &quot;Product Sans&quot;;
border-radius: 10px;
background-color: rgb(255, 239, 115);


</string>
    </property>
    <property name="text">
     <string>Save Key</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 15pt &quot;Product Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>350</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 15pt &quot;Product Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 15pt &quot;Product Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Plaintext File</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 15pt &quot;Product Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Ciphertext</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="encryptButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>680</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 500 15pt &quot;Product Sans&quot;;
border-radius: 10px;
background-color: rgb(255, 239, 115);


</string>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decryptButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>680</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 500 15pt &quot;Product Sans&quot;;
border-radius: 10px;
background-color: rgb(255, 239, 115);


</string>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>440</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 15pt &quot;Product Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Plaintext</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>400</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 15pt &quot;Product Sans&quot;;</string>
    </property>
    <property name="text">
     <string>Ciphertext File</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>340</y>
      <width>20</width>
      <height>381</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>1001</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="loadPlaintext">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>400</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 500 10pt &quot;Product Sans&quot;;
border-radius: 10px;
background-color: rgb(255, 239, 115);


</string>
    </property>
    <property name="text">
     <string>Load File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadCiphertext">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>400</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 500 10pt &quot;Product Sans&quot;;
border-radius: 10px;
background-color: rgb(255, 239, 115);


</string>
    </property>
    <property name="text">
     <string>Load File</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="dKey">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>711</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="eKey">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>711</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="nKey">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>821</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="plaintextName">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>400</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="ciphertextName">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>400</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>640</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 15pt &quot;Product Sans&quot;;
color: rgb(116, 116, 118);</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="cipherResult">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>481</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainResult">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>480</y>
      <width>481</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
